test/cypress/integration/Highcharts/annotations/ellipse/ellipse.spec.js

```diff
@@ -0,0 +1,122 @@
+describe('Stock tools', () => {
+    beforeEach(() => {
+        cy.viewport(1000, 500);
+    });
+
+    before(() => {
+        cy.visit('/stock/demo/stock-tools-gui');
+    });
+
+    it('Should create Ellipse annotation', () => {
```

Comment:
I will try to write some guidance about the cypress but I would be more descriptive here.
Additionally please add the GH issue number.